turtleList: { names : [pturtle, mturtle],
              launchMode : { pturtle: turtlebot,
                             mturtle: turtlebot }, #can be gazebo, turtlebot
              initialGoals: {pturtle : [1.0,1.0,0.0], 
                             mturtle:[1.0,-1.0,0.0] }, 
              xPIDThet: {pturtle : [1.0,0.0,0.0,0.0], 
                        mturtle:[1.0,0.0,0.0,0.0]} , 
              yPIDThet: {pturtle : [1.0,0.0,0.0,0.0], 
                        mturtle:[1.0,0.0,0.0,0.0]} ,
              xMinMax: {pturtle : [-2.0,2.0], 
                        mturtle:[-2.0,2.0] }, 
              yMinMax: {pturtle : [-2.0,2.0], 
                        mturtle:[-2.0,2.0]  }  ,
              queueSize: 10,
              distanceTolerance : 0.05}
